crates/env/src/call/create_builder.rs

```diff
@@ -121,12 +121,31 @@ where
     /// Instantiates the contract and returns its account ID back to the caller.
     #[inline]
     pub fn instantiate(&self) -> Result<R, crate::Error> {
-        crate::instantiate_contract(self).map(FromAccountId::from_account_id)
+        crate::instantiate_contract(self)
+            .map(|inner| {
+                inner.unwrap_or_else(|error| {
+                    panic!("Received a `LangError` while instatiating: {:?}", error)
```

Comment:
```suggestion
                    panic!("Received a `LangError` while instantiating: {:?}", error)
```
Worth mentioning too in the comment that this method will panic on `LangError`, and the alternative of using `try_instantiate`